fcrepo.ttl

```diff
@@ -21,3 +21,8 @@
     rdfs:isDefinedBy : ;
     vs:term_status "working-draft" .
 
+:VersionedResource a rdfs:Resource ;
```

Comment:
Shouldn't this be `rdfs:Class` instead of `rdfs:Resource`? If this is a "type" that a resource (e.g. the LDPCv) would inherit, then it seems like this is more of a class than an individual. And if this is an "interaction model" similar to the LDP interaction models, that would also argue for making this a class.